### PR TITLE
Enhancement/icepak fluent

### PR DESCRIPTION
**Pull request by maxcapodi78**

### CODE REVIEW

File: pyaedt/application/AnalysisIcepak.py [hunk 1]
```diff
@@ -253,7 +254,11 @@ def export_3d_model(self, fileName, filePath, fileFormat=".step", object_list=[]
             allObjects = object_list[:]
 
         self.logger.info("Exporting {} objects".format(len(allObjects)))
-
+        major = -1
+        minor = -1
+        if fileFormat in [".step", ".stp", ".sm3", ".sat", ".sab"]:
+            major = 29
+            minor = 0
```
Comment (MaxJPRey) [inline pyaedt/application/AnalysisIcepak.py:261]: @maxcapodi78 Could you add a short comment here to explain those values? Maybe it is something common and usual for Icepak user.

File: pyaedt/modules/MaterialLib.py [hunk 2]
```diff
@@ -54,6 +54,36 @@ def __getitem__(self, item):
...
+        """
+        mats = []
+        for el, val in self.material_keys.items():
+            if val.thermal_material_type == "Fluid" and val.mass_density.value and float(val.mass_density.value) >= 100:
+                mats.append(el)
+        return mats
+
+    @property
+    def gases(self):
+        """Return the gas materials. A gas is considered a fluid with density lower than 100Kg/m3.
```
Comment (MaxJPRey) [inline pyaedt/modules/MaterialLib.py:74]:
```suggestion
        """Return the gas materials. A fluid is considered to be gas if its density is lower than 100Kg/m3.
```

File: pyaedt/modules/MaterialLib.py [hunk 3]
```diff
@@ -54,6 +54,36 @@ def __getitem__(self, item):
         elif item in list(self.surface_material_keys.keys()):
             return self.surface_material_keys[item]
 
+    @property
+    def liquids(self):
+        """Return the fluids materials. A liquid is considered a fluid with density greather than 100Kg/m3.
```
Comment (MaxJPRey) [inline pyaedt/modules/MaterialLib.py:59]:
```suggestion
        """Return the liquids materials. A liquid is a fluid with density greater than 100Kg/m3.
```

Comment (MaxJPRey) [reply pyaedt/modules/MaterialLib.py:59]:
```suggestion
        """Return the liquids materials. A liquid is considered a fluid with density greater than 100Kg/m3.
```

File: pyaedt/icepak.py [hunk 4]
```diff
@@ -2331,3 +2340,142 @@ def delete_pcb_component(self, comp_name):
 
         self.modeler.oeditor.Delete(arg)
         return True
+
+    @aedt_exception_handler
+    def get_liquid_objects(self):
+        """Return the fluids materials objects.
```
Comment (MaxJPRey) [inline pyaedt/icepak.py:2346]:
```suggestion
        """Return the liquid materials objects.
```

File: pyaedt/modules/MeshIcepak.py [hunk 5]
```diff
@@ -306,6 +306,52 @@ def assign_mesh_level(self, mesh_order, meshop_name=None):
...
+        Parameters
+        ----------
+        objects : list
+            List of objects to which apply the mesh file.
+        filename :  str
+            Full path to .msh file.
+        filename :  str
+            Full path to .msh file.
+        meshop_name :  str, optional
+            Name of the mesh operations
```
Comment (MaxJPRey) [inline pyaedt/modules/MeshIcepak.py:322]:
```suggestion
            Name of the mesh operations. Default is ``None``.
```

File: pyaedt/modules/MaterialLib.py [hunk 6]
```diff
@@ -54,6 +54,36 @@ def __getitem__(self, item):
...
+        """
+        mats = []
+        for el, val in self.material_keys.items():
+            if val.thermal_material_type == "Fluid" and val.mass_density.value and float(val.mass_density.value) >= 100:
+                mats.append(el)
+        return mats
+
+    @property
+    def gases(self):
+        """Return the gas materials. A gas is considered a fluid with density lower than 100Kg/m3.
```
Comment (MaxJPRey) [inline pyaedt/modules/MaterialLib.py:74]:
```suggestion
        """Return the gas materials. A gas is a fluid with density lower than 100Kg/m3.
```

File: pyaedt/modules/MaterialLib.py [hunk 7]
```diff
@@ -54,6 +54,36 @@ def __getitem__(self, item):
         elif item in list(self.surface_material_keys.keys()):
             return self.surface_material_keys[item]
 
+    @property
+    def liquids(self):
+        """Return the liquids materials. A liquid is considered a fluid with density greater than 100Kg/m3.
```
Comment (MaxJPRey) [inline pyaedt/modules/MaterialLib.py:59]:
```suggestion
        """Return the liquids materials. A liquid is a fluid with density greater than 100Kg/m3.
```

File: pyaedt/modules/MeshIcepak.py [hunk 8]
```diff
@@ -306,6 +306,52 @@ def assign_mesh_level(self, mesh_order, meshop_name=None):
...
+        """Assign a mesh from file to objects.
+
+        Parameters
+        ----------
+        objects : list
+            List of objects to which apply the mesh file.
+        filename :  str
+            Full path to .msh file.
+        filename :  str
+            Full path to .msh file.
```
Comment (MaxJPRey) [inline pyaedt/modules/MeshIcepak.py:320]:
```suggestion
```

File: pyaedt/icepak.py [hunk 9]
```diff
@@ -2331,3 +2340,142 @@ def delete_pcb_component(self, comp_name):
...
+            List of all Gas objects.
+        """
+        mats = []
+        for el in self.materials.gases:
+            mats.extend(self.modeler.convert_to_selections(self.modeler.get_objects_by_material(el), True))
+        return mats
+
+    @aedt_exception_handler
+    def generate_fluent_mesh(self, object_lists=None):
+        """Generate a Fluent Mesh for selected object list and assign automatically to the objects.
```
Comment (MaxJPRey) [inline pyaedt/icepak.py:2374]:
```suggestion
        """Generate a Fluent Mesh for selected objects list and assign it automatically to the objects.
```

File: pyaedt/application/AnalysisIcepak.py (hunk 1 above)
Comment (maxcapodi78) [reply pyaedt/application/AnalysisIcepak.py:262]: Done

**Review by MaxJPRey — APPROVED** [review]

**Pull request merged**